CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -118,9 +101,45 @@ Describe "Single compute node protocol tests with utils" {
             -TenantName $ControllerConfig.DefaultProject `
             -Name $NetworkName `
             -Subnet "$( $Subnet.IpPrefix )/$( $Subnet.IpPrefixLen )"
+
+        Write-Host "Creating containers"
+        $Container1ID, $Container2ID = Invoke-Command -Session $Session -ScriptBlock {
+            docker run --network $Using:NetworkName -d iis-tcptest ping -t localhost
```

Comment:
[Change] In case of 'iis-tcptest' I believe 'ping -t localhost' is not required.